api/src/main/java/jakarta/data/page/CursoredPage.java

```diff
@@ -117,34 +125,33 @@
  *                                         {@code PageRequest<Customer>} pageRequest);
  * </pre>
  *
- * <p>Queries that are used with keyset pagination must return entities
- * because keyset cursors are created from the entity attribute values that
- * form the keyset.</p>
+ * <p>Only queries which return entities may be used with key-based pagination
+ * because key-based cursors are created from the entity attribute values that
```

Comment:
```suggestion
 * <p>Only queries which return entities may be used with cursor-based pagination
 * because cursors are created from the entity attribute values that
```